packages/dev/docs/src/docs.css

```diff
@@ -447,14 +447,25 @@ h2.sectionHeader {
 .pageHeader {
   position: fixed;
   top: 0;
-  left: 0;
-  display: flex;
+  display: inline-flex;
   align-items: center;
   justify-content: space-between;
   padding: 0 8px;
   box-sizing: border-box;
   height: var(--spectrum-global-dimension-size-600);
-  width: 100%;
+  [dir='ltr'] & {
+    right: 0;
+  }
+  [dir='rtl'] & {
+    left: 0;
+  }
+}
+
+.themeSwitcher {
```

Comment:
Why do we have this? Can we call it as "empty on purpose because..." if we need to keep it?